niri-config/src/lib.rs

```diff
@@ -1057,6 +1059,10 @@ impl From<niri_ipc::Action> for Action {
             niri_ipc::Action::MoveColumnRight => Self::MoveColumnRight,
             niri_ipc::Action::MoveColumnToFirst => Self::MoveColumnToFirst,
             niri_ipc::Action::MoveColumnToLast => Self::MoveColumnToLast,
+            niri_ipc::Action::MoveColumnLeftOrToMonitorLeft => Self::MoveColumnLeftOrToMonitorLeft,
+            niri_ipc::Action::MoveColumnRightOrToMonitorRight => {
```

Comment:
niri_ipc::Action::MoveColumnRightOrToMonitorRight => Self::MoveColumnRightOrToMonitorRight,

Reply:
this does not follow rustfmt